[26-Nov-2020 10:04:19 Europe/Paris] Uncaught exception : 'Error' Message : 'Using $this when not in object context'
Stack trace : #0 D:\WIP\P5_blog\www\src\Controllers\Book.php(31): Core\HTTPResponse::renderTemplate('Frontend/blog.h...', Array)
#1 [internal function]: Blog\Controllers\Book->indexAction()
#2 D:\WIP\P5_blog\www\Core\Controller.php(65): call_user_func_array(Array, Array)
#3 D:\WIP\P5_blog\www\Core\Router.php(143): Core\Controller->__call('index', Array)
#4 D:\WIP\P5_blog\www\public\index.php(30): Core\Router->dispatch(Object(Core\HTTPRequest))
#5 {main}
Thrown in 'D:\WIP\P5_blog\www\Core\HTTPResponse.php' on line 39
[26-Nov-2020 10:04:19 Europe/Paris] PHP Deprecated:  Non-static method Core\HTTPResponse::renderTemplate() should not be called statically in D:\WIP\P5_blog\www\Core\Error.php on line 52
[26-Nov-2020 10:04:19 Europe/Paris] PHP Fatal error:  Uncaught Error: Using $this when not in object context in D:\WIP\P5_blog\www\Core\HTTPResponse.php:39
Stack trace:
#0 D:\WIP\P5_blog\www\Core\Error.php(52): Core\HTTPResponse::renderTemplate('Errors/500.html...')
#1 [internal function]: Core\Error::exceptionHandler(Object(Error))
#2 {main}
  thrown in D:\WIP\P5_blog\www\Core\HTTPResponse.php on line 39
[26-Nov-2020 10:06:08 Europe/Paris] Uncaught exception : 'Exception' Message : 'No route matched'
Stack trace : #0 D:\WIP\P5_blog\www\public\index.php(30): Core\Router->dispatch(Object(Core\HTTPRequest))
#1 {main}
Thrown in 'D:\WIP\P5_blog\www\Core\Router.php' on line 154
[26-Nov-2020 10:06:08 Europe/Paris] PHP Deprecated:  Non-static method Core\HTTPResponse::renderTemplate() should not be called statically in D:\WIP\P5_blog\www\Core\Error.php on line 52
[26-Nov-2020 10:06:08 Europe/Paris] PHP Fatal error:  Uncaught Error: Using $this when not in object context in D:\WIP\P5_blog\www\Core\HTTPResponse.php:39
Stack trace:
#0 D:\WIP\P5_blog\www\Core\Error.php(52): Core\HTTPResponse::renderTemplate('Errors/404.html...')
#1 [internal function]: Core\Error::exceptionHandler(Object(Exception))
#2 {main}
  thrown in D:\WIP\P5_blog\www\Core\HTTPResponse.php on line 39
[26-Nov-2020 10:08:51 Europe/Paris] Uncaught exception : 'Exception' Message : 'No route matched'
Stack trace : #0 D:\WIP\P5_blog\www\public\index.php(30): Core\Router->dispatch(Object(Core\HTTPRequest))
#1 {main}
Thrown in 'D:\WIP\P5_blog\www\Core\Router.php' on line 154
